text/1000-migrations.rst

```diff
@@ -254,8 +268,14 @@ While the migration block is active:
 * The ``DESCRIBE CURRENT MIGRATION AS JSON`` statement returns a complete
   description of the current migration: statements that have already been
   recorded to be part of the migration script as well as automatically
-  generated list of statements required to complete the migration.  See
-  the "DESCRIBE MIGRATION" section below for details.
+  generated proposal for the next DDL statement to include to advance
+  the migration.  See the "DESCRIBE MIGRATION" section below for details.
+
+* The ``ALTER CURRENT MIGRATION REJECT PROPOSED`` statement is used to
+  inform the server that the currently proposed statement returned by
+  a prior call to ``DESCRIBE CURRENT MIGRATION AS JSON`` is not acceptable
+  and that the next execution of ``DESCRIBE`` should return an alternative
+  proposal.
```

Comment:
What happens if the space of proposals gets exhausted? Do we allow users to manually enter some DDL?

Reply:
Yes